wit/ephemeral/key-value.wit

```diff
@@ -0,0 +1,12 @@
+// General purpose error.
+enum error {
+    success,
+    error,
+}
```

Comment:
I copied this from the `redis-types` types. But why would you need `Error::Success?`

Reply:
Oh yeah, this was left over from older versions of WITX without `expected`.
We should probably update all interfaces to remove the `success` from error enums.